src/apiAction.js

```diff
@@ -99,8 +100,7 @@ const dispatchAction = async (dispatch, { debugRequests, ...options }) => {
   const { baseUrl, path, parentId, prefix, requestAttributes } = options
 
   dispatch({ parentId, type: `${prefix}_REQUEST`, ...requestAttributes })
-
-  const url = `${baseUrl}${path}`
+  const url = `${replace(baseUrl, /\/$/, '')}${path}`
```

Comment:
This way we fix the case that the `/` it is on `baseUrl`, `path` (path build by croods has `/`, but user can set it too.) or missing `/` in both.
```suggestion
  const url = replace(`${baseUrl}/${path}`, /[\/]{2,}/, '/')
```

What do you think?

Reply:
Nice!